Makefile

```diff
@@ -8,6 +8,7 @@ IMAGE_REPO     ?= kedacore
 IMAGE_CONTROLLER = $(IMAGE_REGISTRY)/$(IMAGE_REPO)/keda:$(IMAGE_TAG)
 IMAGE_ADAPTER    = $(IMAGE_REGISTRY)/$(IMAGE_REPO)/keda-metrics-adapter:$(IMAGE_TAG)
 
+VERSION	?= 1.1.0
```

Comment:
VERSION basically equals IMAGE_TAG variable, so would be better to avoid setting the same value on two places.

Reply:
Good point!